### DISCUSSION TITLE
Enhancement: Option for -extent to use virtual canvas information

### DISCUSSION THREAD

**Discussion by fmw42** [ImageMagick/ImageMagick#2895] [Development]
Dirk and I were looking into one of my PSD layer issues and concluded that it would be useful at times if one could have -extent WxH access the virtual canvas offsets. This could be done either with +extent or via -define extent:use-virtual-canvas=on|true. I am ok with either or some other method that makes sense.

**Comment by urban-warrior** [comment]
We need more detail and a use case before we could considering supporting this enhancement.
